packages/dashboard/src/layout/GoldenLayout.scss

```diff
@@ -1,4 +1,4 @@
-@import '../../../components/scss/custom.scss';
+@import '@deephaven/components/scss/custom.scss';
 @import '../../../golden-layout/src/scss/goldenlayout-base.scss';
 @import '../../../golden-layout/src/scss/goldenlayout-dark-theme.scss';
```

Comment:
Any reason we don't have the `golden-layout` package embedding it's scss as well? Or even `dashboard`? It looks like this works, just curious where the line was drawn/why.

Reply:
Fixed these imports

I tried to make it similar to the eslint rule that prevents you from importing outside the source of the package. So any relative path that would have passed through the `packages` dir should be avoided